src/pipedream/components/__init__.py

```
@@ -2,3 +2,33 @@
 
 # isort: skip_file
 
+import typing
+from importlib import import_module
+
+if typing.TYPE_CHECKING:
+    from .types import ComponentsListRequestRegistry
+_dynamic_imports: typing.Dict[str, str] = {"ComponentsListRequestRegistry": ".types"}
+
+
+def __getattr__(attr_name: str) -> typing.Any:
```

Ruff (0.14.8)

Dynamically typed expressions (typing.Any) are disallowed in `__getattr__`

(ANN401)